python_modules/libraries/dagster-dbt/dagster_dbt_tests/test_asset_decorator.py

```diff
@@ -561,3 +567,76 @@ def downstream_of_dbt():
     assert len(downstream_of_dbt.input_names) == 2
     assert downstream_of_dbt.op.ins["orders"].dagster_type.is_nothing
     assert downstream_of_dbt.op.ins["customized_staging_payments"].dagster_type.is_nothing
+
+
+def test_dbt_with_python_interleaving() -> None:
```

Comment:
Can we define `my_dbt_assets` and `python_augmented_customers` with legitimate computations so that we can execute the subset job?